internal/output/html.go

```diff
@@ -91,8 +103,23 @@ type HTMLVulnCount struct {
 	UnFixed  int
 }
 
+type LayerInfo struct {
```

Comment:
rename to HTMLLayerInfo?

Also can you move this above HTMLVulnCount, since that's a field in this.